scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -1052,30 +1051,36 @@
                 (reconcile-jobs conn)
                 (reconcile-tasks (db conn) driver framework-id fenzo)))))
 
+;; Unfortunately, clj-time.core/millis only accepts ints, not longs.
+;; The Period class has a constructor that accepts "long milliseconds",
+;; but since that isn't exposed through the clj-time API, we have to call it directly.
+(defn millis->period
```

Comment:
@wyegelwel - Found the problem! I didn't realize that Clojure was dynamically coercing my longs to ints...

Reply:
Nitpick: we can make this function private.